great_expectations/jupyter_ux/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 import pandas as pd
 import tzlocal
-from IPython.core.display import HTML, display
+from IPython.display import HTML, display
 from packaging import version
```

Comment:
Set set a strict pin so there is no variability - this new import is what is supported with the version we use.